base/src/main/java/com/austindroids/austinfeedsme/di/modules/ActivityBindingModule.kt

```diff
@@ -16,17 +16,17 @@
  * When Dagger.Android annotation processor runs it will create 4 subcomponents for us.
  */
 @Module
-public abstract class ActivityBindingModule {
+abstract class ActivityBindingModule {
     @ActivityScoped
-    @ContributesAndroidInjector(modules = EventsPresenterModule.class)
-    abstract EventsActivity eventsActivity();
+    @ContributesAndroidInjector(modules = [EventsPresenterModule::class])
+    internal abstract fun eventsActivity(): EventsActivity
```

Comment:
`internal` is not needed here on any of these methods, it's better as `public` tbh.

Reply:
i think this happened on converstion - good call out

Reply:
You're interoping with Java, it's a wonder that `internal` works with Dagger at all.